scripts/ci/changelog/templates/host_functions.md.tera

```diff
@@ -1,23 +1,32 @@
 {%- import "change.md.tera" as m_c -%}
 {%- set_global host_fn_count = 0 -%}
+{%- set_global upgrade_first = 0 -%}
 
 {% for pr in changes | sort(attribute="merged_at") -%}
 
 {%- if pr.meta.B and pr.meta.B.B0 -%}
 {#- We skip silent ones -#}
 {%- else -%}
-    {%- if pr.meta.E and pr.meta.E.E4 -%}
+    {%- if pr.meta.E and pr.meta.E.E3 -%}
         {%- set_global host_fn_count = host_fn_count + 1 -%}
             - {{ m_c::change(c=pr) }}
         {% endif -%}
+    {%- elif pr.meta.E and pr.meta.E.E4 -%}
+        {%- set_global upgrade_first = upgrade_first + 1 -%}
+            - {{ m_c::change(c=pr) }}
+        {% endif -%}
     {% endif -%}
 {%- endfor -%}
 
 <!-- {{ host_fn_count }} host functions were detected -->
 
 ## Host functions
 
-{%- if host_fn_count == 0 %}
+{%- if upgrade_first != 0 %}
+⚠️ There is a runtime change that will require nodes to be update BEFORE the runtime upgrade.  
+
+⚠️ It is critical that you update your client before the chain switches to the new runtime.
+{% elif host_fn_count == 0 %}
```

Comment:
There is currently no rule preventing from having E3 and E4 together, this is fine IMO.

I would suggest to have 2 independant sections, one for E3 saying on a "note" level that there is a new host fn and one for E4 which is basically the big warning we have already.

This way we cover all the cases:
- E3 alone
- E3 + E4 together
- E4 alone

Reply:
How does this look? I've separated the sections and changed the host function section to use ℹ️ instead of ⚠️